src/t8_forest/t8_forest_partition.cxx

```diff
@@ -634,9 +633,9 @@ t8_forest_partition_fill_buffer (t8_forest_t forest_from, char **send_buffer, in
     tree_info_pos += sizeof (t8_forest_partition_tree_info_t);
     /* We can now fill the send buffer with all elements of that tree */
     if (num_elements_send > 0) {
-      pfirst_element = t8_element_array_index_locidx (&tree->elements, first_tree_element);
+      const t8_element_t *pfirst_element = t8_element_array_index_locidx (&tree->elements, first_tree_element);
       elem_size = t8_element_array_get_size (&tree->elements);
-      memcpy (*send_buffer + element_pos, pfirst_element, num_elements_send * elem_size);
+      memcpy (*send_buffer + element_pos, (void *) pfirst_element, num_elements_send * elem_size);
```

Comment:
```suggestion
      memcpy (*send_buffer + element_pos, (const void *) pfirst_element, num_elements_send * elem_size);
```
Although it makes no effective difference, this at least shows that we do not cast away the constness. memcpy allows for const src pointer, in contrast to sc_array_copy

Reply:
good point!